jax/interpreters/pxla.py

```diff
@@ -621,13 +624,15 @@ def xla_pmap_impl(fun: lu.WrappedFun, *args, backend, axis_name, axis_size,
                                    in_axes, out_axes_thunk,
                                    donated_invars, global_arg_shapes,
                                    *abstract_args)
+
   # Don't re-abstractify args unless logging is enabled for performance.
   if config.jax_distributed_debug:
     distributed_debug_log(("Running pmapped function", name),
                           ("python function", fun.f),
                           ("devices", devices),
-                          ("abstract args", map(xla.abstractify, args)))
-  return compiled_fun(*args)
+                          ("abstract args", map(xla.abstractify, args)),
+                          ("Fingerprint", compiled_fun[1].fingerprint))
```

Comment:
Nit pick: the other logged fields are all lowercase, so make "fingerprint" all lowercase too :)